### PR TITLE
Devops 198

### PR DESCRIPTION
**Pull request by cwsatpolymath**
Polymesh operator overview and detailed guides:

* Converted to markdown
* Split overview and detailed guides into separate documents
* Added references to precompiled releases and containers
* Removed `--sentry` and `--sentry-nodes` parameters in favour of `--reserved-nodes` and `--reserved-only`
* Reorganised the detailed guide sections

### CODE REVIEW

**Comment by cwsatpolymath** [comment]
Notes:

* You can see the rendered versions at https://github.com/PolymathNetwork/polymesh-tools/tree/DEVOPS-198/docs/operator/overview and https://github.com/PolymathNetwork/polymesh-tools/tree/DEVOPS-198/docs/operator/detailed_guide
* The guides link to each other on the assumption that they are committed to the `main` branch, and as a result some links will show as `404`s until merged.

**Review by maxsam4 — COMMENTED** [review]
Should we add a note somewhere that `--sentry` related flags are deprecated and should not be used?

File: docs/operator/detailed_guide/README.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
+# Polymesh Operator - Detailed Guide
+
+### **Strictly Confidential**
```
Comment (satyamakgec) [inline docs/operator/detailed_guide/README.md:3]: This is a public repo then what is a need for  **Strictly Confidential**

File: docs/operator/detailed_guide/README.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+To run a polymesh node we recommend that you make use of the following options:
+
+* `--name <name>` (optional but recommended): Human-readable name of the nodes that is reported to the telemetry services
+* `--pruning archive`: Ensure that the node maintains a full copy of the blockchain
+* `--base-path <path>` (optional): Specify where Polymesh will look for its DB files and keystore
+* `--db-cache <cache size in MiB>` (optional):  Improve the performance of the polymesh process by increasing its
+  in-memory cache above the default `128` MiB.  On a node with 8GB RAM available a reasonable value is in the
+  ballpark of `4096`.
+
```
[the remark this answers is not in the document]
Comment (cwsatpolymath) [reply docs/operator/detailed_guide/README.md:116]: I would leave the `--wasm-execution compiled` change for a future change once we have done the burn-in on our end.

File: docs/operator/detailed_guide/README.md (hunk 1 above)
Comment (cwsatpolymath) [reply docs/operator/detailed_guide/README.md:3]: I copied this section verbatim from the document shared on Teams.  I will reach out to Nick to review the legalese.

File: docs/operator/overview/README.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
+# Polymesh Operator - Overview
+
+### **Strictly Confidential**
```
Comment (quinndiggitypolymath) [inline docs/operator/overview/README.md:3]: just another of https://github.com/PolymathNetwork/polymesh-tools/pull/4/files#r554869279

**Comment by cwsatpolymath** [comment]
Updated the operator guide back to a single document.  After reviewing I noticed that we were mixing and duplicating topics across both of them.  The new single document is at https://github.com/PolymathNetwork/polymesh-tools/tree/DEVOPS-198/docs/operator#glossary . 

I also reordered the sections to be more consistent with the order in which they should be read.

**Review by adamdossa — APPROVED** [review]

**Review by ndrc81 — APPROVED** [review]

**Pull request merged**